### ISSUE TITLE
Join firmware and mesh-wide-upgrade menus

### ISSUE THREAD

**Issue by javierbrk** [libremesh/lime-app#435]
![imagen](https://github.com/user-attachments/assets/f0baf9cc-2217-4b06-8865-05684994eb7a)
Is it possible to join firmware and mesh-wide-upgrade menus in the main list ? so the main list has only one option and then the user can chose between local upgrade or mesh wide upgrade ?

**Comment by selankon** [comment]
No, is not possible. Atm the current menu implementation doesn't support nested menus. 

We could implement it, or otherwise, implement a page to select one or other.  

In my opinion we could let as is for this iteration and implement nested menus on the future. Maybe @nicoechaniz could give some opinion about this UX
